cfme/tests/infrastructure/test_host_drift_analysis.py

```diff
@@ -49,8 +49,14 @@ def test_host_drift_analysis(appliance, request, setup_provider, provider, host,
     """
     host_collection = appliance.collections.hosts
     test_host = host_collection.instantiate(name=host['name'], provider=provider)
-    wait_for(lambda: test_host.exists, delay=20, num_sec=120, fail_func=sel.refresh,
-             message="hosts_exists")
+    wait_for(lambda: test_host.exists, delay=20, num_sec=120,
+             fail_func=appliance.server.browser.refresh, message="hosts_exists")
+
+    # tabs changed, hack until configure.tasks is refactored for collections and versioned widgets
+    if appliance.version >= '5.9':
+        destination = 'AllTasks'
+    else:
```

Comment:
You could collapse this into one line to clean it up a bit.

Reply:
you could shut your mouth.

Reply:
done